src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml

```diff
@@ -2,7 +2,7 @@
 
 jobs:
 - job: Source_Build_Create_Tarball
-  container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-20210222183538-031e7d2
+  container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33
```

Comment:
I'm not sure this tag exists.  Looks like we need to upgrade.  I would suggest using mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36

The same comment applies to the dev containers - cc @lbussell

Reply:
Should have checked. Was just going through the tags removing suffixes. I'll update this to fedora 36